webpack.config.js

```diff
@@ -14,16 +14,16 @@ module.exports = {
     rules: [
       {
         enforce: 'pre',
-        test: /\.(js|jsx)$/,
+        test: /\.js$/,
```

Comment:
you don't have any `jsx`-files in your project, so there is no need to check them in the config.